string ssid
---
int32 strength 